### PR TITLE
Convert userprofilespage, userPasswordPage & UserImagePage to react

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

File: src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                        dangerouslySetInnerHTML={createLinkElement({
+                            user: user,
+                            imgUrl: imgUrl,
+                            imageClass: imageClass
+                        })}
+                    />
+                </div>
+                <div className='cardFooter visualCardBox-cardFooter'>
+                    <div className='cardText flex align-items-center'>
+                        <div className='flex-grow' style={{overflow: 'hidden', textoverflow: 'ellipsis'}}>
```
Comment (thornbill) [inline src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx:87]:
```suggestion
                        <div className='flex-grow' style={{overflow: 'hidden', textOverflow: 'ellipsis'}}>
```

File: src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                            {user.Name}
+                        </div>
+                        <div
+                            dangerouslySetInnerHTML={createButtonElement()}
+                        />
+                    </div>
+                    <div className='cardText cardText-secondary'>
+                        {lastSeen != '' ? lastSeen : '&nbsp'}
```
Comment (thornbill) [inline src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx:95]:
```suggestion
                        {lastSeen != '' ? lastSeen : ' '}
```

Comment (grafixeyehero) [reply src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx]: https://github.com/jellyfin/jellyfin-web/pull/2954/commits/f5dd9d1ce2b4bb16437a9d26484a6a06535d26e6

File: src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx (hunk 1 above)
Comment (grafixeyehero) [reply src/components/DashboardComponent/users/userprofiles/UserCardBox.tsx]: https://github.com/jellyfin/jellyfin-web/pull/2954/commits/992946c6f7ed69c076375606fbb00f40f0d7260c

**Comment by thornbill** [comment]
Hey @grafixeyehero this PR is getting really big. If you continue migrating pages could you make them separate PRs? This one might already be a bit much to review well. 😅 Thanks!

**Comment by grafixeyehero** [comment]
> 
> 
> Hey @grafixeyehero this PR is getting really big. If you continue migrating pages could you make them separate PRs? This one might already be a bit much to review well. 😅 Thanks!

@thornbill Ok i will create a new PR

**Comment by grafixeyehero** [comment]
in favour of  https://github.com/jellyfin/jellyfin-web/pull/3025

**Pull request closed without merging**